packages/php-wasm/compile/php/phpwasm-emscripten-library.js

```diff
@@ -264,15 +268,29 @@ const LibraryExample = {
 		if (!command) {
 			return 1;
 		}
-
 		const cmdstr = UTF8ToString(command);
 		if (!cmdstr.length) {
 			return 0;
 		}
 
+		let jsArgsArray = [];
```

Comment:
Let's add a short explanation, e.g.:

```suggestion
		// Extracts an array of CLI arguments that should be passed to `command`.
		// On the C side, the args are expressed as `**char` so we must go read 
		// each of the `argsLength` `*char` pointers and convert the associated data into
		// a JavaScript string.
		let jsArgsArray = [];
```

Reply:
👍 I am not familiar with direct `commit suggestion` here and with the conflicts I fear having errors, can I add these suggestions directly in my next commit ?

Reply:
Sure!